braindecode/datasets/base.py

```diff
@@ -158,10 +168,14 @@ class WindowsDataset(BaseDataset):
     transform : callable | None
         On-the-fly transform applied to a window before it is returned.
     targets_from : str
-        Defines whether targets will be extracted from mne.Epochs metadata or mne.Epochs `misc`
-        channels (time series targets). It can be `metadata` (default) or `channels`.
+        Defines whether targets will be extracted from mne.Epochs metadata or
+        mne.Epochs `misc`
+        channels (time series targets). It can be `metadata` (default) or
+        `channels`.
```

Comment:
```suggestion
        Defines whether targets will be extracted from mne.Epochs metadata or
        mne.Epochs `misc` channels (time series targets). It can be `metadata`
         (default) or `channels`.
```